homeassistant/components/notify/__init__.py

```diff
@@ -80,7 +80,7 @@ def setup(hass, config):
 
         def notify_message(notify_service, call):
             """Handle sending notification message service calls."""
-            message = call.data[ATTR_MESSAGE]
+            message = call.data.get(ATTR_MESSAGE, "")
```

Comment:
I'm also not sure if I agree with the fact that notify platforms have an optional message - it's a notify platform!